c/c++.cpp

```diff
@@ -1,9 +1,6 @@
 #include <iostream>
 
-using namespace std;
-
 int main()
 {
-   cout << "Hello World" << endl;
-   return 0;
```

Comment:
I think it's still best to be explicit with `return 0` here.

Reply:
You can check a 22 years old version of the standard and the examples with main don't have a return statement. https://www.lirmm.fr/~ducour/Doc-objets/ISO+IEC+14882-1998.pdf Let's stick to wikipedia as well ? https://en.wikipedia.org/wiki/C%2B%2B#Standardization